core/domain/stats_services.py

```diff
@@ -459,9 +459,8 @@ def _get_calc_output(exploration_id, state_name, calculation_id):
     """Get state answers calculation output domain object obtained from
     StateAnswersCalcOutputModel instance stored in the data store. The
     calculation ID comes from the name of the calculation class used to compute
-    aggregate data from submitted user answers.
-    If 'exploration_version' is VERSION_ALL, this will return aggregated
-    output for all versions of the specified state and exploration.
+    aggregate data from submitted user answers This returns aggregated output
```

Comment:
Nit: missing period at end of sentence, after "answers".

Reply:
Done.